Eat_and_Greet/src/screens/PostEvent/index.tsx

```diff
@@ -22,6 +22,20 @@ export default function PostEvent() {
   let inputs = [title, category, eventDate, startTime, endTime];
   const inputsAsString = ["Title", "Category", "Event Date", "Start Time", "End Time"];
 
+
+  const getNumberOfEventsData = async (numberOfEventsSnapshot : QuerySnapshot): Promise<number[]> => {
```

Comment:
This function is a little funky currently because there should only be one occurance in number of events, so its just storing it in a list for data conversion reasons. Could try to clean, looking for suggestions.